clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/providers/TitusClusterProvider.groovy

```diff
@@ -22,13 +22,17 @@ import com.netflix.spinnaker.cats.cache.CacheData
 import com.netflix.spinnaker.cats.cache.CacheFilter
 import com.netflix.spinnaker.cats.cache.RelationshipCacheFilter
 import com.netflix.spinnaker.clouddriver.core.provider.agent.ExternalHealthProvider
+import com.netflix.spinnaker.clouddriver.docker.registry.provider.DockerRegistryProviderUtils
+import com.netflix.spinnaker.clouddriver.docker.registry.cache.Keys as DockerRegistryCacheKeys
```

Comment:
are there tabs in here or something?

Reply:
no tabs here `import com.netflix.spinnaker.clouddriver.docker.registry.cache.Keys as DockerRegistryCacheKeys` , not sure why its showing up this way

Reply:
Github always renders aliased imports like this. It's indeed odd.